applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -422,6 +430,12 @@ static int initialize(le_audio_receive_cb recv_cb)
 		}
 #endif /* (CONFIG_BT_VCS) */
 
+		ret = ble_mcs_client_init();
```

Comment:
If MCS client is disabled, then we will get an error here. Should this be encoded in an #if as well?

Reply:
I wrote it so that if CONFIG_BT_MCC is n, you get a LOG_ERR("MCC not enabled") and -ECANCELED returned. This is to tell the customer that they need MCC, or they have to go in and change all code related to it.